src/Admin/AbstractAdmin.php

```diff
@@ -284,7 +284,7 @@
     /**
      * The current request object.
      *
-     * @var \Symfony\Component\HttpFoundation\Request
+     * @var \Symfony\Component\HttpFoundation\Request|null
```

Comment:
Please don't use FCQN here.

Reply:
Not my fault but ok

Reply:
I know, but you touched this line, so you could do some cleanup as well :)

Reply:
I could, and did, wait for the push :P